src/functional/ffront/type_info.py

```diff
@@ -117,7 +117,7 @@ def _as_field(dtype: ts.ScalarType, path: tuple[int, ...]) -> ts.FieldType:
                 # The structure of the scan passes argument and the requested
                 # argument type differ. As such we can not extract the dimensions
                 # and just return a generic field shown in the error later on.
-                return ts.FieldType(dims=..., dtype=dtype)
+                return ts.FieldType(dims=[], dtype=dtype)
```

Comment:
```suggestion
                # TODO: we want some generic field type here, but our type system does not support it yet.
                return ts.FieldType(dims=Dimension("..."), dtype=dtype)
```
How about this one? Or pick another name that 1) signifies this can be any field type 2) does not collide with a user picked one easily.